app/assets/stylesheets/application.scss

```diff
@@ -34,23 +38,116 @@ body {
 
 .enter-question {
   padding: 0;
+  background-color: #000DBA;
+  border-color: #007095;
+  color: #6FFDFF;
 }
 
 .main-content-container {
   padding: 0 15px;
 }
 
-input[type=text].postfix.round,
-input[type=email].postfix.round {
-  border-radius: 1000px 0 0 1000px;
-  padding: 0;
+.background {
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+  background: url(asset-path("footer-eight.jpg")) no-repeat;
+  -webkit-background-size: contain;
+  -moz-background-size: contain;
+  -o-background-size: contain;
+  background-size: contain;
+  overflow: auto;
+  background-position: center bottom;
+  max-width: 720px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.frontpage-form {
+  input[type=text] {
+    border-radius: 1000px 0 0 1000px;
```

Comment:
Properties should be ordered background, border, border, border-radius, color, height, line-height, padding